src/retrace/retrace_worker.py

```diff
@@ -243,7 +243,7 @@ def read_package_file(self, crashdir):
             with open(os.path.join(crashdir, "package"), "r") as package_file:
                 crash_package = package_file.read(ALLOWED_FILES["package"])
         except Exception as ex:
-            loging.error("Unable to read crash package from 'package' file: %s" % ex)
+            logging.error("Unable to read crash package from 'package' file: %s" % ex)
```

Comment:
Would it not be more consistent to use `log_error`?

Reply:
Yes, thank you. :+1: